src/Nethermind/Nethermind.Evm/EvmState.cs

```diff
@@ -178,6 +181,7 @@ private int[] RentReturnStack()
                 _accessedAddresses = stateForAccessLists._accessedAddresses;
                 _accessedStorageCells = stateForAccessLists._accessedStorageCells;
                 _destroyList = stateForAccessLists._destroyList;
+                _createList = stateForAccessLists._createList;
```

Comment:
I see you are not creating/restoring snapshot on `_createList`, while using parent collection. This can cause issues as created items will bubble up to parent context?

Reply:
I want it to bubble up. I want a list of created contracts in the whole transaction not only the current context